src/validators/time-after-or-equal.js

```diff
@@ -0,0 +1,54 @@
+import always from 'ramda/src/always';
+import moment from 'moment';
+import _ from 'lodash';
+
+const NOW = 'now';
```

Comment:
minor, I think the value should use upper case too
I've misunderstood this line when reading for the first time (because I thought NOW value should be `NOW` too)
```
ruleObj.params[0].toLowerCase() === NOW
```